### ISSUE TITLE
Media module & i18n

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew-drupal#936]
The media module doesn't seem to support i18n, so I opened this bug:
  https://drupal.org/node/2027605

So how are existing Drupal WET sites using the Media module for images?

**Comment by RobJohnston** [comment]
Hi @mgifford, is this the situation that you experience:  https://github.com/wet-boew/wet-boew-drupal/issues/927.  If so, I've made some progress on another site but am not ready to write it all down yet here... off the top of my head, it involves enabling "File" as a translatable entity type and making File type fields translatable.  Don't even get me started on migration with this thing!

**Comment by mgifford** [comment]
Fun to be working with you on this in both issue queues.  Would be great if we could standardize this with Drupal WET - https://drupal.org/node/2017647#comment-7577367

**Comment by sylus** [comment]
Hey this is exactly has @RobJohnston described we did this for Open Data File has to be enabled as a translatable entity type. Additionally the fields inside of the file entity type (such as images) have to have their fields enabled such as alt and title. Once this is enabled, everything should work. I will work on getting this enabled in the distro by default.

**Comment by mgifford** [comment]
Searching the issue queue, I didn't see any other references for "Open Data File" - can I have a link?

Great to hear you're going to work on enabling this by default in the distro. Thanks!

**Comment by RobJohnston** [comment]
@mgifford, I think he's missing some punctuation.  Read it as, "This is exactly as @RobJohnston described.  We did this for Open Data.  File has to be enabled as a translatable entity type."  The link being data.gc.ca?

It's not an intuitive process so enabling the correct flow as a distro default would be great.

**Comment by mgifford** [comment]
Damn punctuation. Thanks for the clarification.

**Comment by supermoos** [comment]
Did any of you find a way to enable alt and title translations? I have enabled the alt & title fields as translatable under the file types. And under /admin/config/regional/entity_translation, I've also tried both enabling and disabling field translation of the image field on my content type - seems to make no difference. The only thing that seems weird is that under my node content type edit form and the "Synchronize translations"-tab, no fields are available for synchronization?
